design/components/card.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+import { styled } from './stitches.config';
+
+const StyledCard = styled('section', {
+  display: 'inline-block',
+  img: {
+    height: '100%',
+    width: '100%',
+    objectFit: 'cover',
+    transition: `transform .6s ease`,
+  },
+  outline: 'none',
+  position: 'relative',
+
+  'a::after': { position: 'absolute', top: 0, left: 0, right: 0, bottom: 0 },
+
+  '.card-img-container': {
+    height: 324,
+    width: 406,
```

Comment:
Still a width / height.

Reply:
removing
<img width="1260" alt="Screenshot 2020-10-09 at 13 51 55" src="https://user-images.githubusercontent.com/37541400/95586883-99895b80-0a41-11eb-8b4e-87de30baa4ac.png">
 the width and height messed up the style 
<img width="1260" alt="Screenshot 2020-10-09 at 13 51 47" src="https://user-images.githubusercontent.com/37541400/95586843-87a7b880-0a41-11eb-938e-e81160bcd846.png">

Reply:
Can we fix this by setting the content container (white area) to a width of 90%?

If it's too much hassle leave the width / height. We can fix this when we implement it inside a grid view.

Reply:
setting the width to 90% did the trick, but the height is not the same depending on the content

Reply:
setting it to 90% done